src/app/dtos/service.report.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsArray, IsString } from 'class-validator';
+
+/**
+ * Please note: this structure is expected to be used across all services
```

Comment:
Seems like this belongs in protocol-common then.

Reply:
Agreed.  Once we get protocol-common finished I will make a new PR with the updates.